acceptance/tests/api-gateway/api_gateway_test.go

```diff
@@ -283,6 +283,257 @@ func TestAPIGateway_Basic(t *testing.T) {
 	}
 }
 
+func TestAPIGateway_JWTAuth_Basic(t *testing.T) {
+	t.Skip()
+	ctx := suite.Environment().DefaultContext(t)
+	cfg := suite.Config()
+
+	if !cfg.EnableEnterprise {
+		t.Skipf("skipping this test because -enable-enterprise is not set")
+	}
+
+	helmValues := map[string]string{
+		"connectInject.enabled":                       "true",
+		"connectInject.consulNamespaces.mirroringK8S": "true",
+		"global.acls.manageSystemACLs":                "true", // acls must be enabled for JWT auth to take place
+		"global.tls.enabled":                          "true",
+		"global.logLevel":                             "trace",
+	}
+
+	releaseName := helpers.RandomName()
+	consulCluster := consul.NewHelmCluster(t, helmValues, ctx, cfg, releaseName)
+
+	consulCluster.Create(t)
+
+	// Override the default proxy config settings for this test
+	consulClient, _ := consulCluster.SetupConsulClient(t, true)
+	_, _, err := consulClient.ConfigEntries().Set(&api.ProxyConfigEntry{
+		Kind: api.ProxyDefaults,
+		Name: api.ProxyConfigGlobal,
+		Config: map[string]interface{}{
+			"protocol": "http",
+		},
+	}, nil)
+	require.NoError(t, err)
+
+	logger.Log(t, "creating api-gateway resources")
+	out, err := k8s.RunKubectlAndGetOutputE(t, ctx.KubectlOptions(t), "apply", "-k", "../fixtures/cases/api-gateways/jwt-auth")
+	require.NoError(t, err, out)
+	helpers.Cleanup(t, cfg.NoCleanupOnFailure, cfg.NoCleanup, func() {
+		// Ignore errors here because if the test ran as expected
+		// the custom resources will have been deleted.
+		k8s.RunKubectlAndGetOutputE(t, ctx.KubectlOptions(t), "delete", "-k", "../fixtures/cases/api-gateways/jwt-auth")
+	})
+
+	// Create certificate secret, we do this separately since
+	// applying the secret will make an invalid certificate that breaks other tests
+	logger.Log(t, "creating certificate secret")
+	out, err = k8s.RunKubectlAndGetOutputE(t, ctx.KubectlOptions(t), "apply", "-f", "../fixtures/bases/api-gateway/certificate.yaml")
```

Comment:
`.../bases/...` is soooo close to `.../cases/...` that I was almost certain I'd found a typo 🤣